preview/README.md

```diff
@@ -0,0 +1,10 @@
+.NET CLI Preview Docker Image
```

Comment:
What do you think about mentioning the other Dockerfile repository?  Something like 'See dotnet/dotnet-docker for officially released versions.'